.github/workflows/ci_cd.yml

```
@@ -204,8 +204,7 @@ jobs:
       #     retention-days: 7
       
       - name: Deploy Documentation
```

```suggestion
      - name: Deploy Stable Documentation
```
it is better to change the name to stable docs deploy and dev docs deploy, to differentiate faster